code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -215,6 +215,17 @@
 /obj/item/ammo_storage/magazine/m380auto/practice/empty
 	starting_ammo = 0
 
+/obj/item/ammo_storage/magazine/m380auto/extended
+	name = "magazine (.380AUTO extended) "
+	desc = "A magazine designed for .380AUTO vectors. Holds 20 rounds. This one is too big to be outfited into glocks."
```

Comment:
\>outfited

Reply:
uh, should I just fix the grammatical error or pick another word?

Reply:
i think 'to fit' would work better